Firestore/core/src/model/value_util.h

```diff
@@ -67,6 +68,14 @@ void SortFields(google_firestore_v1_ArrayValue& value);
 util::ComparisonResult Compare(const google_firestore_v1_Value& left,
                                const google_firestore_v1_Value& right);
 
+util::ComparisonResult MaxValue(
+    const absl::optional<google_firestore_v1_Value>& left,
+    const absl::optional<google_firestore_v1_Value>& right);
+
+const absl::optional<google_firestore_v1_Value>* MinValue(
```

Comment:
These names clash with the already existing `MinValue`. Can you update the names to reflect that there are doing a comparison?